scripts/run_radar_tests

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 # clone a radar repo
-git clone https://github.com/zendesk/radar.git
+git clone -b pobrien/radar_test_changes_for_message_protocol_v2 git@github.com:zendesk/radar.git
```

Comment:
this should test against master. We are changing no behavior, so tests should pass against master radar.